_includes/schedule.md

```diff
@@ -37,11 +37,17 @@
                 }
             },
             "courseMode": ["online", "synchronous"],
+            "courseWorkload": "90-minutes long cohort call",
```

Comment:
I looked at the spec for a CourseInstance: https://bioschemas.org/profiles/CourseInstance/1.0-RELEASE
I do not see the `courseWorkload`

Reply:
But we are missing some minimum properties like:
- `@id` 
- `location`

Reply:
I found this. I think it got added recently...

> Summary of Changes
Changes since previous draft 0.7 of the CourseInstance profile:
Inserted CourseInstance->courseWorkLoad (it was missing)

Probably why it exists [courseWork](https://schema.org/courseWorkload) on schema.org. 💭